examples/QtSiteConfig/main.py

```diff
@@ -25,6 +25,19 @@ def test():
         # Suppress 'Qt.QtCore' imported but unused warning
         QtCore
 
+    # Test _compatibility_members is applied correctly
+    title = 'Test Widget'
+    from Qt import QtWidgets, QtCompat
+    app = QtWidgets.QApplication(sys.argv)
```

Comment:
You can also supress it like this, I presume:

    _ = QtWidgets.QApplication(sys.argv)